src/CsvHelper/CsvParser.cs

```diff
@@ -160,6 +160,16 @@ public string[] Record
 		/// <param name="configuration">The configuration.</param>
 		public CsvParser(TextReader reader, CsvConfiguration configuration)
 		{
+			if (reader is null)
+			{
+				throw new ArgumentNullException(nameof(reader));
+			}
+
+			if (configuration is null)

```

Comment:
Waht about simplifying configuration null-check also? Null coalescing operator `??` is really great!

Reply:
I'm not against it, but ReSharper detected this style as being the most common in the codebase and went with it so just trying to stay consistent, but happy to go either way.

Reply:
Sometimes someone may become the one who breaks the common practice. 
But of course  - it's all up to the Maintainer, all my comments are just suggestions.